### DISCUSSION TITLE
Install prnMsg not displayed

### DISCUSSION THREAD

**Discussion by apmuthu** [timschofield/webERP#172] [Problems / Bugs] [answered]
The `install/index.php` file makes an initial check for the `config.php` file and is expected to exit after displaying a message. The `function prnMsg()` merely appends the message to an array to be displayed at the end of the script, but the very next statement is `exit;` which makes sure that the script gets truncated thereat.

Hence, replace line 156 of `install/index.php`:
````
		prnMsg(_('It seems that the system has been already installed. If you want to install again, please remove the config.php file first'),'error');
````
with
````
		echo _('It seems that the system has been already installed. If you want to install again, please remove the config.php file first');
````
Furthermore, if the database exists and no tables exist in it, then, the script should be allowed to skip over db creation and `config.php` creation and proceed to database table creation.

**Comment by timschofield** [comment]
I have done the change on the error message, and pushed the commit.

I will need to think how to do the other change you suggest.

Tim

**Comment by timschofield** [comment]
@andrewcouling thinking about this there are a number of places in the installer where prnMsg() is called, and as header.php and footer.php are never used then the messages won't get displayed using the new system. Is it worth having a prnMsgInstaller() function?

**Comment by apmuthu** [comment, marked as answer]
All `prnMsg` have been replaced with `echo` statements in the `install/index.php` file.
Diff and changed files attached.
[PrnMsg_echo_diff.zip](https://github.com/timschofield/webERP-svn/files/1866172/PrnMsg_echo_diff.zip)

**Comment by timschofield** [comment]
Fixes applied, many thanks.

tim
